kafka-ui-react-app/src/components/Connect/List/List.tsx

```diff
@@ -20,6 +20,7 @@ export interface ListProps {
   areConnectorsFetching: boolean;
   connectors: FullConnectorInfo[];
   connects: Connect[];
+  failed: FullConnectorInfo[];
```

Comment:
Connectors or connects?

Reply:
The kafka component is called Kafka Connect and consists of connectors :)